### ISSUE TITLE
unity-base/hud-14.10_p0_p01_p20151012 build failure

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#129]
I wanted to rebuild hud, but I’ve got this build error:

https://gist.githubusercontent.com/and3k/0dedd12c9633aa7aaca2/raw/15886db9c0a91c1edd75269e4fd2a32cc8282adf/hud-14.10_p0_p01_p20151012_build_log

I already remerged libcolumbus, but that didn’t help…

Thanks!
Bela

emerge --info:

```
Portage 2.2.26 (python 3.5.1-final-0, default/linux/amd64/13.0, gcc-5.3.0, glibc-2.22-r1, 3.18.2-gentoo-bela x86_64)
=================================================================
                         System Settings
=================================================================
System uname: Linux-3.18.2-gentoo-bela-x86_64-Intel-R-_Core-TM-_i7-4771_CPU_@_3.50GHz-with-gentoo-2.2
KiB Mem:    32624796 total,  28041840 free
KiB Swap:   35839996 total,  35839996 free
Timestamp of repository gentoo: Sat, 09 Jan 2016 21:30:01 +0000
sh bash 4.3_p42-r1
ld GNU ld (Gentoo 2.25.1 p1.1) 2.25.1
app-shells/bash:          4.3_p42-r1::gentoo
dev-java/java-config:     2.2.0::gentoo
dev-lang/perl:            5.22.1::gentoo
dev-lang/python:          2.7.11-r2::gentoo, 3.3.5-r7::gentoo, 3.4.3-r7::gentoo, 3.5.1-r2::gentoo
dev-util/cmake:           3.4.1::gentoo
dev-util/pkgconfig:       0.28-r3::gentoo
sys-apps/baselayout:      2.2::gentoo
sys-apps/openrc:          0.19.1::gentoo
sys-apps/sandbox:         2.10-r1::gentoo
sys-devel/autoconf:       2.13::gentoo, 2.69-r1::gentoo
sys-devel/automake:       1.11.6-r2::gentoo, 1.12.6-r1::gentoo, 1.13.4-r1::gentoo, 1.14.1-r1::gentoo, 1.15-r1::gentoo
sys-devel/binutils:       2.25.1-r1::gentoo
sys-devel/gcc:            5.3.0::gentoo
sys-devel/gcc-config:     1.8::gentoo
sys-devel/libtool:        2.4.6-r1::gentoo
sys-devel/make:           4.1-r1::gentoo
sys-kernel/linux-headers: 4.3::gentoo (virtual/os-headers)
sys-libs/glibc:           2.22-r1::gentoo
Repositories:

gentoo
    location: /usr/portage
    sync-type: rsync
    sync-uri: rsync://rsync.gentoo.org/gentoo-portage
    priority: -1000

haskell
    location: /var/lib/layman/haskell
    sync-type: laymansync
    sync-uri: git://github.com/gentoo-haskell/gentoo-haskell.git
    masters: gentoo
    priority: 50

jorgicio
    location: /var/lib/layman/jorgicio
    sync-type: laymansync
    sync-uri: https://github.com/jorgicio/jorgicio-gentoo.git
    masters: gentoo
    priority: 50

science
    location: /var/lib/layman/science
    sync-type: laymansync
    sync-uri: git://anongit.gentoo.org/proj/sci.git
    masters: gentoo
    priority: 50

unity-gentoo
    location: /var/lib/layman/unity-gentoo
    sync-type: laymansync
    sync-uri: git://github.com/shiznix/unity-gentoo.git
    masters: gentoo
    priority: 50

and3k-sunrise
    location: /var/lib/overlays/and3k-sunrise
    masters: gentoo
    priority: 9999

ACCEPT_KEYWORDS="amd64 ~amd64"
ACCEPT_LICENSE="*"
CBUILD="x86_64-pc-linux-gnu"
CFLAGS="-march=native -O2 -pipe"
CHOST="x86_64-pc-linux-gnu"
CONFIG_PROTECT="/etc /usr/share/config /usr/share/gnupg/qualified.txt"
CONFIG_PROTECT_MASK="/etc/ca-certificates.conf /etc/dconf /etc/env.d /etc/fonts/fonts.conf /etc/gconf /etc/gentoo-release /etc/revdep-rebuild /etc/sandbox.d /etc/terminfo /etc/texmf/language.dat.d /etc/texmf/language.def.d /etc/texmf/updmap.d /etc/texmf/web2c"
CXXFLAGS="-march=native -O2 -pipe"
DISTDIR="/usr/portage/distfiles"
EMERGE_DEFAULT_OPTS="--jobs --load-average 8 --autounmask n"
FCFLAGS="-O2 -pipe"
FEATURES="assume-digests binpkg-logs config-protect-if-modified distlocks ebuild-locks fixlafiles merge-sync news parallel-fetch preserve-libs protect-owned sandbox sfperms strict unknown-features-warn unmerge-logs unmerge-orphans userfetch userpriv usersandbox usersync xattr"
FFLAGS="-O2 -pipe"
GENTOO_MIRRORS="http://distfiles.gentoo.org"
LANG="en_GB.utf8"
LDFLAGS="-Wl,-O1 -Wl,--as-needed"
MAKEOPTS="-j8 -l8"
PKGDIR="/usr/portage/packages"
PORTAGE_CONFIGROOT="/"
PORTAGE_RSYNC_OPTS="--recursive --links --safe-links --perms --times --omit-dir-times --compress --force --whole-file --delete --stats --human-readable --timeout=180 --exclude=/distfiles --exclude=/local --exclude=/packages"
PORTAGE_TMPDIR="/var/tmp"
USE="X a52 aac acl acpi alsa amd64 autoipd avahi ayatana bash-completion berkdb bidi blas blast bluetooth bookmarks branding bzip2 cairo cdda cjk clang cli colord conductor contrib cracklib crypt cups cxx dbus declarative dri dts dvd dvdr ebook eds emboss encode equalizer evo exif expat fam ffmpeg firefox flac fortran frei0r gcj gdbm gif glamor gnome gnome-keyring gnome-online-accounts gpg gpm gstreamer gtk hdf5 iconv icu inotify introspection ipv6 java jpeg kdenlive lapack lcms ldap libnotify libsecret mad matplotlib mdnsresponder-compat melt mmx mmxext mng modules mp3 mp4 mpeg multilib nautilus nbconvert ncurses networkmanager nls notebook nptl ntfs ogg opengl openmp pam pango parcheck pcre pdf png policykit postproc ppds printsupport pulseaudio python qt3support qt4 qt5 readline sdl seccomp semantic-desktop session spell sse sse2 sse3 sse4_1 ssl ssse3 staging startup-notification svg sync-plugin-portage systemd tcpd test-programs theora tiff tinfo truetype tty-helpers udev udisks unicode upnp upower usb utils v4l vaapi video vorbis webkit widgets wifi wmf wxwidgets x264 xattr xcb xetex xkb xml xmp xv xvfb xvid zeitgeist zeroconf zlib" ABI_X86="64 32" ALSA_CARDS="ali5451 als4000 atiixp atiixp-modem bt87x ca0106 cmipci emu10k1x ens1370 ens1371 es1938 es1968 fm801 hda-intel intel8x0 intel8x0m maestro3 trident usb-audio via82xx via82xx-modem ymfpci" APACHE2_MODULES="authn_core authz_core socache_shmcb unixd actions alias auth_basic authn_alias authn_anon authn_dbm authn_default authn_file authz_dbm authz_default authz_groupfile authz_host authz_owner authz_user autoindex cache cgi cgid dav dav_fs dav_lock deflate dir disk_cache env expires ext_filter file_cache filter headers include info log_config logio mem_cache mime mime_magic negotiation rewrite setenvif speling status unique_id userdir usertrack vhost_alias" CALLIGRA_FEATURES="kexi words flow plan sheets stage tables krita karbon braindump author" CAMERAS="ptp2" COLLECTD_PLUGINS="df interface irq load memory rrdtool swap syslog" CPU_FLAGS_X86="aes avx avx2 fma3 mmx mmxext popcnt sse sse2 sse3 sse4_1 sse4_2 ssse3" ELIBC="glibc" GPSD_PROTOCOLS="ashtech aivdm earthmate evermore fv18 garmin garmintxt gpsclock itrax mtk3301 nmea ntrip navcom oceanserver oldstyle oncore rtcm104v2 rtcm104v3 sirf superstar2 timing tsip tripmate tnt ublox ubx" GRUB_PLATFORMS="efi-64" INPUT_DEVICES="keyboard mouse evdev" KERNEL="linux" LCD_DEVICES="bayrad cfontz cfontz633 glk hd44780 lb216 lcdm001 mtxorb ncurses text" LIBREOFFICE_EXTENSIONS="presenter-console presenter-minimizer" LINGUAS="en en_GB de de_AT" OFFICE_IMPLEMENTATION="libreoffice" PHP_TARGETS="php5-5" PYTHON_SINGLE_TARGET="python2_7" PYTHON_TARGETS="python2_7 python3_3 python3_4 python3_5" RUBY_TARGETS="ruby20 ruby21 ruby22 ruby23" USERLAND="GNU" VIDEO_CARDS="intel" XTABLES_ADDONS="quota2 psd pknock lscan length2 ipv4options ipset ipp2p iface geoip fuzzy condition tee tarpit sysrq steal rawnat logmark ipmark dhcpmac delude chaos account"
Unset:  CC, CPPFLAGS, CTARGET, CXX, INSTALL_MASK, LC_ALL, PORTAGE_BUNZIP2_COMMAND, PORTAGE_COMPRESS, PORTAGE_COMPRESS_FLAGS, PORTAGE_RSYNC_EXTRA_OPTS, USE_PYTHON

=================================================================
                        Package Settings
=================================================================

unity-base/hud-14.10_p0_p01_p20151012::unity-gentoo was built with the following:
USE="-test" ABI_X86="64" PYTHON_TARGETS="python2_7"
```

**Comment by shiznix** [comment]
Definitely part of gcc-5 moving over to the new C++11 ABI.
See here under the 'Troubleshooting' section -> https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html

**Comment by and3k** [comment]
yup, but I rebuild the whole system already and that didn’t help, so has to be more than that…?

**Comment by shiznix** [comment]
Should be fixed in commit f74b68472ababb8e9a2d5dc62175f4c79c8a98d2

**Comment by and3k** [comment]
This now builds. Thanks!
